dashboard/app/controllers/plc/tasks_controller.rb

```diff
@@ -8,7 +8,7 @@ def show
 
```

Comment:
I'm surprised it wasn't preselected.  It looks like `.pluck` doesn't work if the `@task` hasn't been saved yet.  I'll update to use `.map` for the `TaskController#new`.